File: next/content/problems/asm-working-hours/solutions/scala/starter.scala
object Solution {
  def solution(queries: Seq[Seq[String]]): Seq[String] = {
    val workers = scala.collection.mutable.Map.empty[String, Any]
    val result = scala.collection.mutable.ArrayBuffer.empty[String]

    for (query <- queries) query.head match {
      case "ADD_WORKER" => result += ""
      case "REGISTER" => result += ""
      case "GET" => result += ""
      case "TOP_N_WORKERS" => result += ""
      case "PROMOTE" => result += ""
      case "CALC_SALARY" => result += ""
      case "SET_DOUBLE_PAY" => result += ""
      case _ => result += ""
    }

    workers.clear()
    result.toSeq
  }
}

File: next/content/problems/asm-working-hours/solutions/go/starter.go
package solution

func Solution(queries [][]string) []string {
	workers := map[string]any{}
	result := []string{}

	for _, query := range queries {
		switch query[0] {
		case "ADD_WORKER":
			result = append(result, "")
		case "REGISTER":
			result = append(result, "")
		case "GET":
			result = append(result, "")
		case "TOP_N_WORKERS":
			result = append(result, "")
		case "PROMOTE":
			result = append(result, "")
		case "CALC_SALARY":
			result = append(result, "")
		case "SET_DOUBLE_PAY":
			result = append(result, "")
		default:
			result = append(result, "")
		}
	}

	_ = workers
	return result
}
